api/src/main/java/org/pmiops/workbench/api/WorkspaceController.java

```diff
@@ -100,20 +113,39 @@ private void setCdrVersionId(Workspace workspace,
     }
   }
 
+  private FirecloudWorkspaceId generateFirecloudWorkspaceId(String name) {
```

Comment:
I'm confused. This code implements a 1:1 correspondance between project and workspaces? Doesn't generating a project name imply that a GCP project exists to match it? (I thought these corresponded to billing projects which are GCP projects.)

I'm probably just confused, but I thought we wanted to create FireCloud Workspaces for now within a single GCP project.